src/olympia/lib/settings_base.py

```diff
@@ -1668,3 +1668,7 @@ def get_redis_settings(uri):
 SHELL_PLUS_POST_IMPORTS = (
     ('olympia', 'amo'),
 )
+
+DEFAULT_FXA_CONFIG_NAME = 'default'
+INTERNAL_FXA_CONFIG_NAME = 'internal'
```

Comment:
That means you can remove [the 2 lines that do the same thing from `settings.py`](https://github.com/mozilla/addons-server/blob/223af8742a8a858e14e944225d7dd8b914b829ae/settings.py#L116-L117), right ?